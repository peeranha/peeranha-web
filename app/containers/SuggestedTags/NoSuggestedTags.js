import React from 'react';
import { FormattedMessage } from 'react-intl';

import messages from './messages';

const NoSuggestedTags = () => (
  <div className="text-center">
    <FormattedMessage {...messages.noSuggestedTags} />
  </div>
);

export default NoSuggestedTags;
