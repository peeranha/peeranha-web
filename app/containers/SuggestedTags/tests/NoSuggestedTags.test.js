import NoSuggestedTags from '../NoSuggestedTags';

describe('NoSuggestedTags', () => {
  it('test', () => {
    expect(NoSuggestedTags()).toMatchSnapshot();
  });
});
