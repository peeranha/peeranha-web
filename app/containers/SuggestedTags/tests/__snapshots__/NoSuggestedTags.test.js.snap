// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSuggestedTags test 1`] = `
<div
  className="text-center"
>
  <FormattedMessage
    defaultMessage="There are no suggested tags"
    id="app.containers.SuggestedTags.noSuggestedTags"
    values={Object {}}
  />
</div>
`;
